cityofaustin/atd-moped
# Re-structures: Phases, Sub-phases, Milestones

Closes https://github.com/cityofaustin/atd-data-tech/issues/5865
Closes https://github.com/cityofaustin/atd-data-tech/issues/5863
Closes https://github.com/cityofaustin/atd-data-tech/issues/5864
Closes https://github.com/cityofaustin/atd-data-tech/issues/5930
Closes https://github.com/cityofaustin/atd-data-tech/issues/5846

To test:

```
1. Run locally:  $ ./hasura-cluster restart
2. The migration should be successful and hasura launches without a problem
3. Open moped editor and add, edit and delete phases.
4. Run a roll-back: $ hasura migrate apply --down 1 
```
